### PR TITLE
Voice recognition

### PR DESCRIPTION
**Pull request by Chanell13**

### CODE REVIEW

File: src/app/voicerecognition-sample/voicerecognition-sample.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<h1>Voice Recognition</h1>
+
+<h2>Usage samples</h2>
+<p>VoiceRecongnition is implemented as a directive. Therefore it can extend any input or textarea to allow voice input whenever text is accepted.</p>
```
Comment (Foxandxss) [inline src/app/voicerecognition-sample/voicerecognition-sample.component.html:4]: VoiceRecognition

File: src/app/voicerecognition-sample/voicerecognition-sample.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<p>To use it click on the mic icon and talk.</p>
+<p>Laguage must be configured to improve recognition accuracy.</p>
```
Comment (Foxandxss) [inline src/app/voicerecognition-sample/voicerecognition-sample.component.html:6]: Language

**Review by Foxandxss — CHANGES_REQUESTED** [review]
For me missing:

Warning to say that only works on chrome right now
I would add a dropdown in the demo to select other languages. We can't show the doc in English and the demo in Spanish.

File: src/app/voicerecognition-sample/voicerecognition-sample.component.html (hunk 1 above)
Comment (pjmolina) [reply src/app/voicerecognition-sample/voicerecognition-sample.component.html:4]: Fair points. I will add them.

Comment (pjmolina) [reply src/app/voicerecognition-sample/voicerecognition-sample.component.html]: added

File: src/app/voicerecognition-sample/voicerecognition-sample.component.html (hunk 2 above)
Comment (pjmolina) [reply src/app/voicerecognition-sample/voicerecognition-sample.component.html]: fixed

**Pull request merged**